tests/components/lcn/test_events.py

```diff
@@ -0,0 +1,132 @@
+"""Tests for LCN events."""
+from unittest.mock import patch
+
+from pypck.inputs import Input, ModSendKeysHost, ModStatusAccessControl
+from pypck.lcn_addr import LcnAddr
+from pypck.lcn_defs import AccessControlPeriphery, KeyAction, SendKeyCommand
+
+from homeassistant.components.lcn import host_input_received
```

Comment:
Please don't interact with any integration details in tests of integrations

https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations

Reply:
I initiated this test by using the pypi code.